roles/ogm/tasks/main.yml

```diff
@@ -1,5 +1,15 @@
 ---
 - name: Install and configure OGM backup
   ansible.builtin.include_tasks: script_and_cron.yml
-  when: ogm_enabled
```

Comment:
I'm not sure if this is an improvement: Now `when` statements are added to all individual tasks like 
```
  when:
    - item.prm_location is defined
    - item.server is defined
    - item.user is defined
    - item.psql_dump_location is defined
```
for one task and
  when:
    - item.prm_location is defined
    - item.server is defined
    - item.user is defined
```
for other tasks. This means that for example when `item.psql_dump_location` is missing, some tasks will be applied and some will get skipped resulting in an incomplete backup config; is that the intended behaviour? Or should the play fail when a config is incomplete? And what should happen when for example `item.user` and `item.prm_location` are specified, but `item.server` is omitted?